src/DssSpell.sol

```diff
@@ -19,400 +19,379 @@ pragma solidity 0.8.16;
 }
 
 interface RwaInputConduitLike {
     function mate(address usr) external;
+    function hate(address usr) external;
     function file(bytes32 what, address data) external;
 }
 
 interface RwaOutputConduitLike {
     function file(bytes32 what, address data) external;
     function hope(address usr) external;
+    function nope(address usr) external;
     function mate(address usr) external;
+    function hate(address usr) external;
     function kiss(address who) external;
-}
-
-interface VestLike {
-    function restrict(uint256 _id) external;
-    function create(address _usr, uint256 _tot, uint256 _bgn, uint256 _tau, uint256 _eta, address _mgr) external returns (uint256 id);
-    function yank(uint256 _id) external;
-}
-interface NetworkPaymentAdapterLike {
-    function bufferMax() external view returns (uint256);
-    function minimumPayment() external view returns (uint256);
-    function file(bytes32 what, uint256 data) external;
-    function file(bytes32 what, address data) external;
-}
-
-interface DssCronSequencerLike {
-    function windows(bytes32) external view returns (uint256 start, uint256 length);
-}
-
-interface PoolConfiguratorLike {
-    struct InitReserveInput {
-        address aTokenImpl;
-        address stableDebtTokenImpl;
-        address variableDebtTokenImpl;
-        uint8 underlyingAssetDecimals;
-        address interestRateStrategyAddress;
-        address underlyingAsset;
-        address treasury;
-        address incentivesController;
-        string aTokenName;
-        string aTokenSymbol;
-        string variableDebtTokenName;
-        string variableDebtTokenSymbol;
-        string stableDebtTokenName;
-        string stableDebtTokenSymbol;
-        bytes params;
-    }
-    function initReserves(InitReserveInput[] calldata input) external;
-    function configureReserveAsCollateral(
-        address asset,
-        uint256 ltv,
-        uint256 liquidationThreshold,
-        uint256 liquidationBonus
-    ) external;
-    function setBorrowableInIsolation(address asset, bool borrowable) external;
-    function setDebtCeiling(address asset, uint256 newDebtCeiling) external;
-}
-
-interface AaveOracleLike {
-    function setAssetSources(address[] calldata assets, address[] calldata sources) external;
+    function pick(address who) external;
+    function push() external;
+    function push(uint256 wad) external;
 }
 
 contract DssSpellAction is DssAction {
     // Provides a descriptive tag for bot consumption
     // This should be modified weekly to provide a summary of the actions
-    // Hash: cast keccak -- "$(wget 'https://raw.githubusercontent.com/makerdao/community/ce40e721ba58dc631ee1b66f5259423dd8e504ce/governance/votes/Executive%20vote%20-%20May%2024%2C%202023.md' -q -O - 2>/dev/null)"
+    // Hash: cast keccak -- "$(wget 'https://raw.githubusercontent.com/makerdao/community/master/governance/votes/Executive%20vote%20-%20June%207%2C%202023.md' -q -O - 2>/dev/null)"
```

Comment:
Hash will need to change - new link after updating it based on Arby's feedback - https://github.com/makerdao/community/blob/78ca48d951e0b1a80d233497dbbb8a0ebd63749d/governance/votes/Executive%20vote%20-%20June%2014%2C%202023.md

as an aside the hash should point to the commit not the file